### PR TITLE
FIX: Email templates not compatible with Multicompany

### PR DESCRIPTION
**Pull request by fmarcet**

### CODE REVIEW

File: htdocs/admin/dict.php [hunk 1]
```diff
@@ -154,7 +155,7 @@
 $tabsql[22]= "SELECT rowid   as rowid, code, label, active FROM ".MAIN_DB_PREFIX."c_input_reason";
 $tabsql[23]= "SELECT t.rowid, t.taux, c.label as country, c.code as country_code, t.fk_pays as country_id, t.note, t.active, t.accountancy_code_sell, t.accountancy_code_buy FROM ".MAIN_DB_PREFIX."c_revenuestamp as t, ".MAIN_DB_PREFIX."c_country as c WHERE t.fk_pays=c.rowid";
 $tabsql[24]= "SELECT rowid   as rowid, code, label, active FROM ".MAIN_DB_PREFIX."c_type_resource";
-$tabsql[25]= "SELECT rowid   as rowid, label, type_template, private, position, topic, content, active FROM ".MAIN_DB_PREFIX."c_email_templates";
+$tabsql[25]= "SELECT rowid   as rowid, label, type_template, private, position, topic, content, active FROM ".MAIN_DB_PREFIX."c_email_templates WHERE entity = ".$conf->entity;
```
Comment (hregis) [inline htdocs/admin/dict.php]: @fmarcet can you replace "$conf->entity" with : 

getEntity('email_template', 1)

I will add the ability to share templates between entities in a future version of the module.
thanks

Comment (fmarcet) [inline htdocs/admin/dict.php]: Done!

**Comment by marcosgdf** [comment]
It is a feature request, not a bug :/

Related: #3155

**Comment by fmarcet** [comment]
The problem is not that all templates are displayed in all entities, the problem is that if you create a template in the entity 2, it is saved always with entity = 1 so it can't be used in the entity 2. You can only use email templates in the entity 1, other entities is impossible. For me it is a bug.

File: htdocs/admin/dict.php [hunk 2]
```diff
@@ -154,7 +155,7 @@
 $tabsql[22]= "SELECT rowid   as rowid, code, label, active FROM ".MAIN_DB_PREFIX."c_input_reason";
 $tabsql[23]= "SELECT t.rowid, t.taux, c.label as country, c.code as country_code, t.fk_pays as country_id, t.note, t.active, t.accountancy_code_sell, t.accountancy_code_buy FROM ".MAIN_DB_PREFIX."c_revenuestamp as t, ".MAIN_DB_PREFIX."c_country as c WHERE t.fk_pays=c.rowid";
 $tabsql[24]= "SELECT rowid   as rowid, code, label, active FROM ".MAIN_DB_PREFIX."c_type_resource";
-$tabsql[25]= "SELECT rowid   as rowid, label, type_template, private, position, topic, content, active FROM ".MAIN_DB_PREFIX."c_email_templates";
+$tabsql[25]= "SELECT rowid   as rowid, label, type_template, private, position, topic, content, active FROM ".MAIN_DB_PREFIX."c_email_templates WHERE entity = ".getEntity('email_template',1);
```
Comment (simnandez) [inline htdocs/admin/dict.php:159]: @fmarcet: In a call to getEntity the sql WHERE  xxx = is bad, in this case you must use:
WHERE entity IN ( ".getEntity('email_template',1).")";
Replace the "=" with "IN" and parenthesis.

**Pull request merged**